<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Secondwindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(191, 208, 188)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_deleteadd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid  rgb(170, 85, 0); ;
    border-radius: 50px;
	background-color: rgb(0, 0, 127);
    color: white;
}
QPushButton:hover {
background-color: rgb(49, 54, 127);
    border: 2px solid rgb(0, 0, 127) ;

}
QPushButton:pressed {
    background-color: #1f618d;

}</string>
    </property>
    <property name="text">
     <string>Delete / Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_display">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid  rgb(170, 85, 0); ;
    border-radius: 50px;
	background-color: rgb(0, 0, 127);
    color: white;
}
QPushButton:hover {
background-color: rgb(49, 54, 127);
    border: 2px solid rgb(0, 0, 127) ;

}
QPushButton:pressed {
    background-color: #1f618d;

}</string>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: rgb(0, 0, 127);
    color: rgb(255, 255, 255);
    border: 2px solid rgb(170, 85, 0);
    border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>chose a button</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>551</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border : 2px solid  rgb(170, 85, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pix">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/libraryplanet.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_search">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border: 2px solid  rgb(170, 85, 0); ;
    border-radius: 50px;
	background-color: rgb(0, 0, 127);
    color: white;
}
QPushButton:hover {
background-color: rgb(49, 54, 127);
    border: 2px solid rgb(0, 0, 127) ;

}
QPushButton:pressed {
    background-color: #1f618d;

}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
